docs/source/release.rst

```diff
@@ -7,6 +7,8 @@ Release Notes
    These release notes are for versions of ibis **1.0 and later**. Release
    notes for pre-1.0 versions of ibis can be found at :doc:`/release-pre-1.0`
 
+* :release:`1.2.1 <pending>`
+* :bug:`2009` Change window bound behavior in pandas backend to match other backends
```

Comment:
can you be a bit more specific here; e.g. when specifying ``preceding=`` it did x now does y